package/gluon-respondd/files/lib/gluon/reload.d/009-respondd-stop

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+/etc/init.d/gluon-respondd stop
```

Comment:
Maybe have the respondd scripts at 100/900, and the VPN scripts at 200/800? No need to use the second and third digit with the first scripts we add.